packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "0.25.0",
+  "version": "0.22.0-fb-plate-support-6715.5",
```

Comment:
Don't forget to merge in master from labkey-ui-components before publishing the final release version and merging. Looks like a couple of other PRs have been merged to this repo in the last few days.